docs/how-to/enable-monitoring.md

```diff
@@ -1,14 +1,21 @@
-# Monitoring (COS)
+(enable-monitoring)=
+# How to enable monitoring
+
 The Canonical Observability Stack (COS) is a set of tools that facilitates gathering, processing, visualizing, and setting up alerts on telemetry signals generated by workloads in and outside of Juju.
 
-The etcd charm can use COS to connect to Grafana and Prometheus to use monitoring, alert rules, and log features.
+The etcd charm can use COS to connect to [Grafana](https://grafana.com/) and [Prometheus](https://prometheus.io/) to use monitoring, alert rules, and logging features.
 
 ## Deploy COS
-Deploy the [`cos-lite`](https://charmhub.io/topics/canonical-observability-stack) bundle in a Kubernetes environment following the [MicroK8s guide](https://charmhub.io/topics/canonical-observability-stack/tutorials/install-microk8s). Since the etcd charm is deployed directly on a cloud infrastructure environment, it requires offering the endpoints of the COS relations. The [offers-overlay](https://github.com/canonical/cos-lite-bundle/blob/main/overlays/offers-overlay.yaml) can be used, and this step is shown in the [COS tutorial](https://charmhub.io/topics/canonical-observability-stack/tutorials/install-microk8s#heading--deploy-the-cos-lite-bundle-with-overlays).
 
-Once the COS bundle is deployed, you should have Grafana and Prometheus running in your Kubernetes cluster. Run `juju status` to see the status of the COS model. The output should look similar to the following:
+Deploy the [`cos-lite`](https://charmhub.io/topics/canonical-observability-stack) bundle in a Kubernetes controller following the [MicroK8s guide](https://charmhub.io/topics/canonical-observability-stack/tutorials/install-microk8s). 
```

Comment:
Question: Do we want to mention `cos-lite` but link to COS instead? There is a dedicated link for COS-lite too: https://charmhub.io/topics/canonical-observability-stack/editions/lite